src/citrine/informatics/predictors.py

```diff
@@ -130,4 +127,4 @@ def __str__(self):
 
     def post_build(self, project_id: UUID, data: dict):
         """Creates the predictor report object."""
-        self.report = ReportResource(project_id, self.session).get(data['id'])
+        self.report = lambda: ReportResource(project_id, self.session).get(data['id'])
```

Comment:
This is inconsistent with the rest of the api. Let's fix the underlying problem instead.

Reply:
👍